src/Listener/CodeCoverageListener.php

```diff
@@ -21,6 +21,7 @@
 use SebastianBergmann\CodeCoverage\CodeCoverage;
 use SebastianBergmann\CodeCoverage\Report;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use SebastianBergmann\FileIterator\Facade as FileIteratorFacade;
```

Comment:
Should be ordered alphabetically like the rest of the imports.

Reply:
Ok, I've pushed the change.